src/acl.ts

```diff
@@ -760,6 +760,7 @@ common.util.promisifyAll(Acl, {
   exclude: ['request'],
 });
 
-module.exports = Acl;
-
-module.exports.AclRoleAccessorMethods = AclRoleAccessorMethods;
+export {
+  Acl,
+  AclRoleAccessorMethods,
```

Comment:
`AclRoleAccessorMethods` is only used in tests, will have to fix the import later.